actions/views/root.js

```diff
@@ -80,3 +81,9 @@ export function loadTranslations(locale, url) {
     };
 }
 
+export function loadWarnMetricsStatus() {
```

Comment:
I think this wrapper function is not needed here, at the end of the day, `getWarnMetricsStatus` is already an action, you can import it and use it directly wherever is needed.

Reply:
I removed the `loadWarnMetricsStatus` and directly referenced the `getWarnMetricsStatus` action, thanks for catching it = )